pmsoService/product_flow/features/deliverytrip/api/view_modules/list_create_views.py

```
@@ -39,8 +40,12 @@ def create(self, request, *args, **kwargs):
         serializer = DeliveryTripCreateSerializer(data=request.data)
         serializer.is_valid(raise_exception=True)
         form_data = serializer.to_command()
-        new_trip = create_trip(form_data, request.user)
-        
+        try:
+            new_trip = create_trip(form_data, request.user)
+        except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)